backend/src/entities/note.entity.ts

```diff
@@ -29,6 +29,12 @@ export class Note {
   @Column({ nullable: true })
   date: Date;
 
+  @Column({ nullable: true })
+  testll: string;
+
+  @Column({ nullable: true })
+  msgoop: string;
+
```

Comment:
Please remove your test columns.
Also, remove all generated migration files. And generate one main migration file for all entities. For that, you should first drop your database tables.